app/views/citizens/income_summaries/_add_other_income.html.erb

```diff
@@ -0,0 +1,10 @@
+<li>
+  <h2 class="app-task-list__section">
+            <span class="app-task-list__section-number">
+              <img src="/assets/plus_icon.svg" />
+            </span>
+    <a class="govuk-body" href="<%= citizens_identify_types_of_income_path %>" aria-describedby="eligibility-completed">
```

Comment:
eligibility-completed doesn't seem relevant here. Or am I missing something?
Do we need a `aria-describedby` on this link at all?